nodes/cma/nodes-cma-autoscaling-custom.adoc

```diff
@@ -49,3 +49,21 @@ Successfully updated ScaleTarget
 ----
 
 You can temporarily pause the autoscaling of a workload object, if needed. For example, you could pause autoscaling before performing cluster maintenance.
+
+.Custom metrics autoscaler workflow
+image::564_OpenShift_Custom_Metrics_Autoscaler_0224.png[Custom metrics autoscaler workflow]
+
+1. A user creates or modifies a scaled object custom resource for a workload on the cluster. The object contains the scaling configuration for that workload. Prior to accepting the new object, the OpenShift API server sends it to the custom metrics autoscaler admission webhooks process to ensure that the object is valid. If validation succeeds, the API server persists the object.
+
+2. The custom metrics autoscaler controller watches for new or modified scaled objects. When the OpenShift API server notifies the controller of a change, the controller monitors any external trigger sources (or, data sources) specified in the object for changes to the metrics data. One or more scalers request scaling data from the external trigger source. For example, for a Kafka trigger type, the controller uses the Kafka scaler to communicate with a Kafka instance to obtain the data requested by the trigger.
+
+3. The controller creates a horizontal pod autoscaler object for the scaled object. As a result, the Horizontal Pod Autoscaler (HPA) Operator starts monitoring the scaling data associated with the trigger. The HPA requests scaling data from the cluster OpenShift API server endpoint.
+
```

Comment:
Line 62: Unless "ask" is truly the correct term in this context, I wonder if another term, such as query or retrieve, could be used? So maybe something like "..it uses a GRPC connection to the controller to query for the most recent trigger data..."

Reply:
> Line 62: Unless "ask" is truly the correct term in this context, I wonder if another term, such as query or retrieve, could be used? So maybe something like "..it uses a GRPC connection to the controller to query for the most recent trigger data..."

I agree that `ask` is a form of anthropomorphism -- [see the ISG entry](https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism); `request` would be better.